src/ray/object_manager/plasma/stats_collector.cc

```diff
@@ -168,8 +171,22 @@ void ObjectStatsCollector::OnObjectRefDecreased(const LocalObject &obj) {
   }
 }
 
+int64_t ObjectStatsCollector::GetNumBytesCreatedCurrent() const {
+  return num_bytes_created_by_worker_ + num_bytes_restored_ + num_bytes_received_ +
+         num_bytes_errored_;
+}
+
 void ObjectStatsCollector::RecordMetrics() const {
-  // TODO(sang): Add metrics.
+  ray::stats::STATS_object_store_memory.Record(
+      GetNumBytesCreatedCurrent() - num_bytes_unsealed_,
+      {{"Location", ray::rpc::ObjectLocation_Name(ray::rpc::ObjectLocation::IN_MEMORY)}});
```

Comment:
Are we planning to track "Used"/"Available"? Isn't the sum of these metrics supposed to be the total memory?

Reply:
Yes, I guess this will be in a follow-up for 2.2 maybe. More breakdowns of the usage.